src/components/Filters/ToggleFilter.tsx

```diff
@@ -3,36 +3,45 @@ import { Filter } from "src/components/Filters/types";
 import { Switch } from "src/inputs/Switch";
 import { TestIds } from "src/utils/useTestIds";
 
-export type ToggleFilterProps = { label?: string; enabledValue?: boolean; defaultValue?: boolean };
+export type ToggleFilterProps<V> = {
+  label?: string;
+  onValue?: V | undefined;
+  offValue?: V | undefined;
```

Comment:
This is a breaking change, but doubling down on the "on/off" language instead of enabled/disabled b/c for whatever reason it seems to make things more clear.